File: files/400-499/439/CH-439.py
import pandas as pd

path = "400-499/439/CH-439 Filter.xlsx"
input = pd.read_excel(path, usecols="B:D", nrows=6, skiprows=2)
test = pd.read_excel(path, usecols="H:J", nrows=2, skiprows=2)

result = input.copy()
result = result.loc[lambda x: x["Sales"] > x["Sales"].shift()].reset_index(drop=True)
test.columns = result.columns

print(result.equals(test))
# True

File: files/400-499/439/CH-439.R
library(tidyverse)
library(readxl)

path <- "400-499/439/CH-439 Filter.xlsx"
input <- read_excel(path, range = "B3:D8")
test <- read_excel(path, range = "H3:J5")

result <- input %>%
  filter(Sales > lag(Sales))

all.equal(result, test)
# True
